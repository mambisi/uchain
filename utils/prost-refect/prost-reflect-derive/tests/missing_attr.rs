use prost_reflect_derive::ReflectMessage;

#[derive(ReflectMessage)]
pub struct MyMessage {}

fn main() {}
