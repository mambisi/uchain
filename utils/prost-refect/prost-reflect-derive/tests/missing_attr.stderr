error: missing #[prost_reflect] attribute
 --> tests/missing_attr.rs:4:12
  |
4 | pub struct MyMessage {}
  |            ^^^^^^^^^
